src/DependencyGraph/FormulaCellVertex.ts

```diff
@@ -151,6 +160,11 @@ export class MatrixVertex extends FormulaVertex {
     }
   }
 
+  setNoSpace(): InterpreterValue {
+    this.matrix = new ErroredMatrix(new CellError(ErrorType.REF, ErrorMessage.NoSpaceForArrayResult), MatrixSize.error())
```

Comment:
> Shouldn't this be a SPILL error type?

GS would return REF and that's what I followed. Do we want to introduce new error type at this moment? 

> Can we provide some additional info to the error, as width and height that would allow to display it as shown on the screenshot?

It would be a nice feature indeed but right now we do not store any additional data inside errors except address of a root of a problem.  I would save it for the future.

Reply:
> > Shouldn't this be a SPILL error type?
> 
> GS would return REF and that's what I followed. Do we want to introduce new error type at this moment?

Yes, please. It will be an additional information that helps to differentiate them. ODFF allows vendors to introduce custom errors.
 
> > Can we provide some additional info to the error, as width and height that would allow to display it as shown on the screenshot?
> 
> It would be a nice feature indeed but right now we do not store any additional data inside errors except address of a root of a problem. I would save it for the future.

Ok 👍

Reply:
Done.